pages/api/auth/signup.ts

```diff
@@ -9,14 +9,17 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     return null
   }
 
-  const { username, password } = req.body
+  const { email, password } = JSON.parse(req.body)
 
   const users = client.db().collection('user')
-  const exists = await users.findOne({ username: username })
+  const exists = await users.findOne({ username: email })
   if (exists) {
     res.status(401).json({ error: 'User already exists' })
   } else {
-    const newUser = await users.insertOne(req.body)
+    const newUser = await users.insertOne({
+      username: email,
+      password: password,
```

Comment:
This will change after password encryption, can talk to Dillon and Nick about this at GM tomorrow